src/org/labkey/test/components/domain/RangeValidatorPanel.java

```diff
@@ -0,0 +1,207 @@
+package org.labkey.test.components.domain;
+
+import org.labkey.remoteapi.query.Filter;
+import org.labkey.test.Locator;
+import org.labkey.test.components.WebDriverComponent;
+import org.labkey.test.components.html.Input;
+import org.labkey.test.components.html.SelectWrapper;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+
+import static org.labkey.test.components.html.Input.Input;
+
+public class RangeValidatorPanel extends WebDriverComponent<RangeValidatorPanel.ElementCache>
+{
+    final WebElement _el;
+    final WebDriver _driver;
+
+    public RangeValidatorPanel(WebElement element, WebDriver driver)
+    {
+        _el = element;
+        _driver = driver;
+    }
+
+    @Override
+    public WebElement getComponentElement()
+    {
+        return _el;
+    }
+
+    @Override
+    public WebDriver getDriver()
+    {
+        return _driver;
+    }
+
+
+    public RangeValidatorPanel setFirstCondition(Filter.Operator operator)
+    {
+        expand();
+        elementCache().firstConditionSelect().selectByValue(operator.getUrlKey());
+        return this;
+    }
+
+    public RangeValidatorPanel setFirstValue(String value)
+    {
+        expand();
+        elementCache().firstFilterValueInput().setValue(value);
+        return this;
+    }
+
+    public RangeValidatorPanel setSecondCondition(Filter.Operator operator)
+    {
+        expand();
+        elementCache().secondConditionSelect().selectByValue(operator.getUrlKey());
+        return this;
+    }
+
+    public RangeValidatorPanel setSecondValue(String value)
+    {
+        expand();
+        elementCache().secondFilterValueInput().setValue(value);
+        return this;
+    }
+
+    public RangeValidatorPanel setDescription(String value)
+    {
+        expand();
+        elementCache().descriptionInput().setValue(value);
+        return this;
+    }
+
+    public RangeValidatorPanel setErrorMessage(String value)
+    {
+        expand();
+        elementCache().errorMessageInput().setValue(value);
+        return this;
+    }
+
+    public RangeValidatorPanel setName(String name)
+    {
+        expand();
+        elementCache().nameInput().setValue(name);
+        return this;
+    }
+
```

Comment:
We should make this work with the existing `FieldDefinition.RangeValidator` class.
e.g.
```
public RangeValidatorPanel addValidator(RangeValidator validator)
```

Reply:
I attempted this, but ran into issues converting RangeValidator and Filter.Operator back and forth.  Using Filter.Operator has distinct advantages- (it is pretty complete, its getUrlKey() function matches to select option values, as well as to URL expressions)
Sure, we could make a lot of older tests work more easily in the short term by not using Filter.Operator- but then we'd end up maintaining both enums, hoping they remain parallel, and selecting options by text and not being able to use the urlKey expressions to predict the expressions as they will appear on the domain field JSON

Reply:
Perhaps the best option here will be to implement setters that take both Filter.Operator and RangeValidator's params, support them both

Reply:
I think I have a better solution: pipe the operator through the RangeType enum that FieldDefinition.RangeValidator uses- and pass that to the existing setter